inclusive-design/wecount.inclusivedesign.ca
# feat: remove COVID Data Monitor, redirect to new URL (fix #1636)

## Description

Removes the map page and COVID Data Monitor dependencies; redirects `/map` to `https://map.wecount.inclusivedesign.ca`.

## Steps to test

1. Visit `/map`

**Expected behavior:** You are redirected to the standalone deploy of the COVID Data Monitor.

## Additional information

Not applicable.

## Related issues

Resolves #1636.
